app/api/v1/endpoints/auth.py

```
@@ -485,54 +486,151 @@ async def refresh_token(
 
 @router.post("/logout")
 async def logout(
-    credentials: Optional[HTTPAuthorizationCredentials] = Depends(security),
+    credentials: Optional[HTTPAuthorizationCredentials] = Depends(HTTPBearer(auto_error=False)),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)